.babelrc

```diff
@@ -1,3 +1,4 @@
 {
-  "presets": ["env","es2015","stage-0"]
+  "presets": ["env","es2015","stage-0"],
+  "sourceMaps": "inline"
```

Comment:
Oh, yeah good point. I don't think this really helped actually. I'll just remove it